core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildRunner.java

```diff
@@ -70,4 +77,41 @@ protected void preBuild(List<String> buildArgs) throws IOException, InterruptedE
     protected void postBuild() throws InterruptedException, IOException {
     }
 
+    /**
+     * Run {@code command} in {@code workingDirectory} and log error if {@code errorMsg} is not null.
+     *
+     * @param command The command to run
+     * @param errorMsg The error message to be printed in case of failure.
+     *        If {@code null} the failure is ignored, but logged.
+     * @param workingDirectory The directory in which to run the command
+     */
+    void runCommand(String[] command, String errorMsg, File workingDirectory) {
+        log.info(String.join(" ", command).replace("$", "\\$"));
+        Process process = null;
+        try {
+            final ProcessBuilder processBuilder = new ProcessBuilder(command);
+            if (workingDirectory != null) {
+                processBuilder.directory(workingDirectory);
+            }
+            process = processBuilder.start();
```

Comment:
Looks like you are you are not redirecting the output streams like `ProcessUtil#launchProcess` was doing

Reply:
Indeed, but that's how things where before (except for cleanupServer).

`runCommand` is used to run `native-image --shutdown-server`, `docker rm container-name` and `docker cp src dest`. IMO it's OK to **not** redirect output for these commands (as discussed in https://github.com/quarkusio/quarkus/pull/15288/files#r582745324). WDYT?

Reply:
Okay

Reply:
Thanks George

Reply:
Thanks for fixing this